processor/src/stack/aux_trace.rs

```diff
@@ -81,3 +81,35 @@ impl<E: FieldElement<BaseField = Felt>> AuxColumnBuilder<E> for AuxTraceBuilder
         }
     }
 }
+
+// OVERFLOW STACK ROW
+// ================================================================================================
+
+/// A single row in the stack overflow stack. Each row contains the following values:
```

Comment:
Is this intentional "stack overflow stack"?

Reply:
Oops - fixed to "stack overflow table"